react-relay/index.d.ts

```diff
@@ -31,6 +31,24 @@ declare module "react-relay" {
         response: any
     }
 
+    type RelayMutationStatus =
+      'UNCOMMITTED' | // Transaction hasn't yet been sent to the server. Transaction can be committed or rolled back.
+      'COMMIT_QUEUED' | // Transaction was committed but another transaction with the same collision key is pending, so the transaction has been queued to send to the server.
+      'COLLISION_COMMIT_FAILED' | //Transaction was queued for commit but another transaction with the same collision key failed. All transactions in the collision queue, including this one, have been failed. Transaction can be recommitted or rolled back.
+      'COMMITTING' | // Transaction is waiting for the server to respond.
+      'COMMIT_FAILED';
+
+    class RelayMutationTransaction {
+      applyOptimistic(): RelayMutationTransaction;
+      commit(): RelayMutationTransaction;
+      recommit(): void;
+      rollback(): void;
+      getError(): Error;
```

Comment:
This is my first time dealing with flow like this. While it's true that the declaration says it can be null, reading the rest of the code it seems this will really only ever be Error or undefined.